code/controlconfig/controlsconfigcommon.cpp

```diff
@@ -178,6 +178,7 @@ void control_config_common_init_bindings() {
 	(CYCLE_NEXT_PRIMARY,                             KEY_PERIOD, -1, WEAPON_TAB, 1, "Cycle Primary Weapon Forward",           CC_TYPE_TRIGGER)
 	(CYCLE_PREV_PRIMARY,                              KEY_COMMA, -1, WEAPON_TAB, 1, "Cycle Primary Weapon Backward",          CC_TYPE_TRIGGER)
 	(CYCLE_PRIMARY_WEAPON_SEQUENCE,                       KEY_O, -1, WEAPON_TAB, 1776, "Cycle Primary Weapon Firing Rate",       CC_TYPE_TRIGGER)
+	(CYCLE_PRIMARY_WEAPON_PATTERN,			  KEY_ALTED | KEY_O, -1, WEAPON_TAB, 0, "Cycle Primary Weapon Firing Pattern",	  CC_TYPE_TRIGGER) // need to get help from someone to track down what the next free number is
```

Comment:
...Well? Did you?

Reply:
localize.cpp::67.  The XSTR's are dynamically assigned now, the XSTR_SIZE define should still be incremented to aid in addition of new label strings.

Note: this does mean we have to check before merging,  I don't know if github will catch collisions on the XSTR_SIZE changes.